### PR TITLE
[INTEROP-7281] Expand OpenShift Pipelines to v1.13 for OCP 4.15 and 4.16.

### PR DESCRIPTION
**Pull request by dfrazzette**
Per [SRVKP-3694](https://issues.redhat.com/browse/SRVKP-3694), Pipelines 1.11 is not supported on OCP 4.15. Expanding the scenario to test Pipelines 1.13 on OCP 4.15.

### CODE REVIEW

**Comment by dfrazzette** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.13-openshift-pipelines-ocp4.15-lp-interop-openshift-pipelines-interop-aws

File: ci-operator/jobs/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.13-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    org: openshift-pipelines
+    repo: release-tests
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: openshift-pipelines-ocp4.16-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.16"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-pipelines-release-tests-release-v1.13-openshift-pipelines-ocp4.16-lp-interop-openshift-pipelines-interop-aws
```
Comment (calebevans) [inline ci-operator/jobs/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.13-periodics.yaml:100]: Can you add the `reporter_config` section here? Should look like [this](https://github.com/openshift/release/blob/2984f584c8c5c6d340a2f6aa4c74e4a8311e13fd/ci-operator/jobs/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11-periodics.yaml#L20-L30).

**Review by calebevans — CHANGES_REQUESTED** [review]
One small thing

**Pull request closed without merging**